pulp/pulp_packaging
# reorganize jjb definitions

This brings our JJB definitions in-line with the layout recommended by
JJB itself. Great care was taken to make no changes in the actual output
of JJB so that this PR can be reasonably tested.

To test:
- check out the parent of this commit
- run `jenkins-jobs test` against jobs defined in this directory, write
  the output xml to a file
- update back to this commit
- run `jenkins-jobs test` against jobs defined in this directory, write
  the output xml to a different file than before
- diff the two files, see that they are the same (If they aren't, let me know)
